rust-toolchain

```diff
@@ -1 +1 @@
-nightly
+nightly-2021-03-16
```

Comment:
Please, leave the toolchain as set in `master` or in case of collision, to `nightly-06-06-2021`.